spec/defines/sftp_jail_jail_spec.rb

```diff
@@ -56,6 +56,28 @@
                                                                            'X11Forwarding'          => 'no'
                                                                          })
       end
+
+      let :params do
+        {
+          user: 'alice',
+          group: 'alice',
+          password_authentication: 'yes'
+        }
+      end
+      let :title do
```

Comment:
maybe this needs to be wrapped in a new context block so title can be overwritten?

Reply:
I tried here: https://github.com/voxpupuli/puppet-sftp_jail/pull/74/commits/6c5633152a8317735a582928c7da5083411b8542 didn't help. :(